test/basic/v128_select.wast

```diff
@@ -0,0 +1,24 @@
+(module
+  (func (export "func1") (result i32)
+    v128.const i64x2 0xaaaaaaaaaaaaaaaa 0xaaaaaaaaaaaaaaaa
+    v128.const i64x2 0xbbbbbbbbbbbbbbbb 0xbbbbbbbbbbbbbbbb
+    i32.const 1
+    select
+    v128.const i64x2 0xaaaaaaaaaaaaaaaa 0xaaaaaaaaaaaaaaaa
+    i64x2.eq
+    i64x2.all_true
```

Comment:
What is the point of these extra steps?
A second select which uses the result of a compare instruction would be a good idea though.

Reply:
There was a bug with returning a `v128`, I didn't want to risk having to debug an unrelated issue, so I worked around it by returning an `i32` instead.  I think the issue might as simple as `v128` not having a string serialization implemented.

Reply:
Are you sure the issue is not introduced by the patch? I think many functions return with v128.

Reply:
I'm sure.

```
% git checkout 'v128-select^'
% git show --no-patch --format=%h HEAD 
ed6d35d
% cat v128.wast                       
(module
(func (export "f") (result v128) (v128.const f64x2 1 0)))
(assert_return (invoke "f") (v128.const f64x2 0 1))
% out/release/x86/walrus v128.wast
Assertion failed at 3: f() expected v128, but got 
RELEASE_ASSERT_NOT_REACHED at /home/pronaip/src/walrus/src/shell/Shell.cpp (584)
[1]    200724 IOT instruction (core dumped)  out/release/x86/walrus v128.wast
```

Reply:
So, it does correctly check what is returned, but it can't print the difference if the return doesn't match the expected value.

Reply:
Then there is no bug with v128, but if the expected differs from the returned value, it cannot print it:
https://github.com/Samsung/walrus/blob/main/src/runtime/Value.h#L368

There is no issue with: `(assert_return (invoke "f") (v128.const f64x2 1 0))`